api/Identity-Provider.yaml

```diff
@@ -0,0 +1,1561 @@
+openapi: 3.1.0
+x-stoplight:
+  id: 6f1syzijynu40
+info:
+  title: Identity Provider
+  version: '1.0'
+  contact:
+    name: MOSIP Team
+    email: info@mosip.io
+    url: 'https://www.mosip.io/'
+  description: |-
+    This API document details on the below categories of endpoints
+    <ul>
+    <li> Management - Endpoints for creation and updation of OIDC client details </li>
+    <li> OIDC - All OIDC compliant endpoints for performing the Open ID Connect flows</li>
+    <li> UI - All endpoints used by the UI application </li>
+    </ul>
+
+    <b>Abbreviations:</b></br></br>
+    OIDC - Open ID Connect</br>
+    IdP - Identity provider</br>
+    PMP - Partner Management portal</br>
+    KYC - Know Your Customer</br>
+    IDA - Authentication server</br>
+    UIN - Unique Identification Number</br>
+    VID - Virtual Identifier</br>
+    PSUT - Partner(Relying Party) Specific User Token</br>
+
+        
+        
+       
+  license:
+    name: MPL-2.0
+    url: 'https://www.mozilla.org/en-US/MPL/2.0/'
+  summary: Open ID Connect based identity provider for large scale authentication
+servers:
+  - url: 'https://dev.mosip.net/v1/idp'
+paths:
+  /client-mgmt/oidc-client:
+    post:
+      summary: Create OIDC Client Endpoint
+      operationId: post-client
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              description: OIDC client details
```

Comment:
Some description attributes are appearing in quotes and some are not - is there an agreed patter to these we should follow?

Reply:
Obviously, multi-line descriptions are using |- which is fine